### PR TITLE
evgen24 - movie-seat-booking init

### PR DESCRIPTION
**Pull request by EvgenKich**
## Movie seat booking by Evgen24

[Demo](https://evgeniy241984.github.io/movie-seat-booking/)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/evgeniy24/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+                  <li>2</li>
+                  <li>3</li>
+                  <li>4</li>
+                  <li>5</li>
+                  <li>6</li>
+                </ul>
+                <ul class="left-sector">
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
```
Comment (A-Ostrovnyy) [inline submissions/evgeniy24/html-movie-seat-booking/index.html]: Add attribute `value` to the input.

File: submissions/evgeniy24/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+                    </label>
+                  </li>
+                  <li>
+                    <label>
+                      <input class="seats-input" type="checkbox" />
```
Comment (A-Ostrovnyy) [inline submissions/evgeniy24/html-movie-seat-booking/index.html]: Add styles on `:focus` state.

File: submissions/evgeniy24/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+                  <li>2</li>
+                  <li>3</li>
+                  <li>4</li>
+                  <li>5</li>
+                  <li>6</li>
+                </ul>
+              </div>
+            </form>
+          </section>
+          <button class="booking__button">BOOK NOW</button>
```
Comment (A-Ostrovnyy) [inline submissions/evgeniy24/html-movie-seat-booking/index.html]: Don't use visual representing in markup. Use CSS for it.

File: submissions/evgeniy24/html-movie-seat-booking/styles/media.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  .seats-input {
+    width: 17px;
+    height: 17px;
+  }
+}
+
+/*Iphones X > landscape */
+@media only screen and (min-width: 812px) and (max-height: 415px) {
```
Comment (A-Ostrovnyy) [inline submissions/evgeniy24/html-movie-seat-booking/styles/media.css]: Avoid adding unused rules.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Extra like for TV Show.
See my comments below!

File: submissions/evgeniy24/html-movie-seat-booking/index.html (hunk 1 above)
Comment (EvgenKich) [reply submissions/evgeniy24/html-movie-seat-booking/index.html]: Done, see changes after a new commit!

File: submissions/evgeniy24/html-movie-seat-booking/index.html (hunk 2 above)
Comment (EvgenKich) [reply submissions/evgeniy24/html-movie-seat-booking/index.html]: Done, see changes after a new commit!

File: submissions/evgeniy24/html-movie-seat-booking/index.html (hunk 3 above)
Comment (EvgenKich) [reply submissions/evgeniy24/html-movie-seat-booking/index.html]: Done, see changes after a new commit!

File: submissions/evgeniy24/html-movie-seat-booking/styles/media.css (hunk 4 above)
Comment (EvgenKich) [reply submissions/evgeniy24/html-movie-seat-booking/styles/media.css]: Done, see changes after a new commit!

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**